docs/_posts/0000-03-04-the-case-for-branching.md

```diff
@@ -277,3 +277,26 @@ What are they
 What do they look like  
 What is the effect  
 How do you resolve them  
+
+--
+
+## Conflict resolution
+
++ Easiest to use a powerful tool
++ IDEs support this
+    + Will offer to do so if you pull down changes using the IDE and conflict detected
+    + Also possible to use the IDE conflict resolution if you `git pull` on the command line
+    + IntelliJ: VCM menu > Git > Resolve Conflicts…
```

Comment:
I'm pretty sure IntelliJ calls it VCS not VCM.

Reply:
Good spot batman!